tests/e2e/pageObjects/DicomViewer.ts

```diff
@@ -28,60 +26,48 @@ export class DicomViewer {
   }
 
-  async login({ username, password }): Promise<void> {
-    await this.page.locator(this.elements.userNameSelector).fill(username)
-    await this.page.locator(this.elements.passwordSelector).fill(password)
-    await this.page.locator(this.elements.loginButtonSelector).click()
-    await this.page.locator(this.elements.webContentSelector).waitFor()
+  async login(user): Promise<void> {
+    const stepUser = await getUser({ user })
+    await global.page.locator(this.elements.userNameSelector).fill(stepUser.displayName)
+    await global.page.locator(this.elements.passwordSelector).fill(stepUser.password)
+    await global.page.locator(this.elements.loginButtonSelector).click()
+    await global.page.locator(this.elements.webContentSelector).waitFor()
   }
 
   async logout(): Promise<void> {
-    await this.page.locator(this.elements.userMenuButtonSelector).click()
-    await this.page.locator(this.elements.logoutSelector).click()
+    await global.page.locator(this.elements.userMenuButtonSelector).click()
+    await global.page.locator(this.elements.logoutSelector).click()
   }
 
   async upload({ filename }): Promise<void> {
-    await this.page.locator(this.elements.resourceUploadButton).click()
-    await this.page
+    await global.page.locator(this.elements.resourceUploadButton).click()
+    await global.page
       .locator(this.elements.fileUploadInput)
       .setInputFiles(`${config.assets}/${filename}`)
-    await this.page.locator(this.elements.uploadInfoCloseButton).click()
-    await this.page.locator(util.format(this.elements.resourceNameSelector, filename)).waitFor()
+    await global.page.locator(this.elements.uploadInfoCloseButton).click()
+    await global.page.locator(util.format(this.elements.resourceNameSelector, filename)).waitFor()
   }
 
   async previewDicomFile({ filename }): Promise<void> {
-    await this.page.locator(util.format(this.elements.resourceNameSelector, filename)).click()
-    await expect(this.page.locator(this.elements.dicomCanvasSelector)).toBeVisible()
-    await expect(
-      this.page.locator(util.format(this.elements.appbarResourceNameSelector, filename))
-    ).toBeVisible()
+    await global.page.locator(util.format(this.elements.resourceNameSelector, filename)).click()
   }
 
-  async checkVIPMetadata(): Promise<void> {
-    await expect(this.page.locator(this.elements.dicomVIPMetadataSelector)).toBeVisible()
-  }
-
-  async checkExtendedMetadata(): Promise<void> {
-    await this.page.locator(this.elements.dicomExtendedMetadataBtnSelector).click()
-    await expect(this.page.locator(this.elements.dicomMetadataSidebarSelector)).toBeVisible()
-  }
-
-  async checkPatientName({ patientName, metadataLocation }): Promise<void> {
-    let actualPatientName
+  async getPatientName(metadataLocation): Promise<void> {
+    let patientName
     if (metadataLocation === 'VIP metadata') {
```

Comment:
I think a switch statement would be a little nicer way to put multiple cases in one single getter function. Or just separate the two cases into separate getter functions, there wouldn't be much repeated code by doing this and I think with clear naming it's easier to understand what the function does.